patches/redirectToDVLogin.js

```diff
@@ -0,0 +1,28 @@
+function redirectToLoginPage() {
+	window.location.pathname = `/${window.location.pathname.split("/")[1]}/LoginEndpoint.aspx`;
+}
+
+window.appendModule({
+    onlyOnReloads: true,
+    run: redirectToLoginPage,
+    doesRunHere: () =>
+        window.location.hostname === "dziennik-uczen.vulcan.net.pl"
+		&& !window.location.pathname.split("/")[2]
+});
+
+const pathSegment = window.location.pathname.split("/")[1];
+
+const newSpanContent = `
+  Nie pamiętasz loginu lub hasła? Możemy wysłać na Twój <nobr>e-mail</nobr> link do zresetowania dostępu. 
+  Przejdź do <a href="https://dziennik-logowanie.vulcan.net.pl/${pathSegment}/PasswordReset/UnlockRequest" class="link-simple">formularza</a> 
+  lub użyj <a href="https://uonetplus.vulcan.net.pl/${pathSegment}" class="link-simple">innej metody logowania</a>
+`;
+
+const targetSpan = document.querySelector('div.flex-col:nth-child(4) > span:nth-child(3)');
```

Comment:
It changes content only on one page, it behaves differently while using/not using patch for displaying both inputs on same page.

Changing content for all the elements `document.querySelectorAll(".input-components ~ .form-gap > span")` returns, should do the job